docs/sphinx/sections/api/apidocs/libraries/dagster-looker.rst

```diff
@@ -8,9 +8,30 @@ your other data assets, and how changes to other data assets might impact your L
 
 .. currentmodule:: dagster_looker
 
+**********
+Looker API
+**********
 
-Assets
-======
+Here, we provide interfaces to manage Looker projects using the Looker API.
+
+Assets (Looker API)
+===================
+
+.. autoclass:: LookerResource
+
+.. autoclass:: DagsterLookerApiTranslator
+
+.. autoclass:: RequestStartPdtBuild
```

Comment:
Should we make these classes experimental before making them public in our docs? Or do we think the APIs are stable?

Reply:
Thanks for the callout!

I'll mark `LookerResource` as experimental, and keep everything else unmarked so we don't have excessive spamming in the logs.